sport/app/football/controllers/ResultsController.scala

```diff
@@ -37,7 +37,7 @@ object ResultsController extends MatchListController with CompetitionResultFilte
   private def page(tag: Option[String] = None): Option[FootballPage] = {
     def allPage = new FootballPage("football/results", "football", "All results", "GFE:Football:automatic:results")
     def competitionPage = (competition: Competition) => new FootballPage(s"${competition.url}/results", "football", s"${competition.fullName} results", "GFE:Football:automatic:competition results")
-    def teamPage = (team: FootballTeam) => new FootballPage(s"/football/$tag/results", "football", s"${tag} results", "GFE:Football:automatic:team results")
+    def teamPage = (team: FootballTeam) => new FootballPage(s"/football/${tag.getOrElse("")}/results", "football", s"${team.name} results", "GFE:Football:automatic:team results")
```

Comment:
What's the significance of not having a tag? That would produce a parameter to `FootballPage` that looks like `/football//results`. Maybe if we don't have a tag we shouldn't render the page?